download-directory/src/main/java/org/reactome/release/downloadDirectory/Main.java

```diff
@@ -27,10 +27,17 @@ public static void main(String[] args) throws Exception {
 
 		logger.info("Beginning Download Directory step");
 		String pathToConfig = "";
+		String pathToStepConfig = "";
 		if (args.length > 0) {
 			pathToConfig = args[0];
+			if (args.length > 1) {
+				pathToStepConfig = args[1];
+			} else {
+				pathToStepConfig = "src/main/resources/stepsToRun.config";
+			}
 		} else {
 			pathToConfig = "src/main/resources/config.properties";
+			pathToStepConfig = "src/main/resources/stepsToRun.config";
```

Comment:
It looks like if there is more than one argument, `pathToStepConfig` will be an empty String.  Is that okay?

Reply:
No, good catch. I'll make the path in the final 'else' the default string.

Reply:
Actually, looking over this again, I think I'm wrong.  Both `pathToConfig` and `pathToStepConfig` should be set.  It might be a good place to use ternaries for the varaibles initialization, though:

```
String pathToConfig = args.length > 0 ? args[0] : Paths.get("src/ /main/resources/config.properties").toString();
String pathToStepConfig = args.length > 1 ? args[1] : Paths.get("src/main/resources/stepsToRun.config").toString();
```

Reply:
Thanks, I used these but I didn't use the `Paths.get().toString()`, I just used the string variable